ibm/provider/provider.go

```diff
@@ -625,6 +625,9 @@ func Provider() *schema.Provider {
 			"ibm_scc_posture_profile":           scc.DataSourceIBMSccPostureProfileDetails(),
 			"ibm_scc_posture_group_profile":     scc.DataSourceIBMSccPostureGroupProfileDetails(),
 			"ibm_scc_posture_scope_correlation": scc.DataSourceIBMSccPostureScopeCorrelation(),
+			"ibm_scc_posture_get_credential":    scc.DataSourceIBMSccPostureGetCredential(),
```

Comment:
Remove the word "get"
Datasource by default indicates they are get or read oeprations
ibm_scc_posture_credential..